src/index.js

```diff
@@ -2,5 +2,5 @@
 
 export {default} from './command'
 export {default as Topic} from './topic'
-export {BooleanFlag, StringFlag, NumberFlag, Flag} from './flags'
+export {boolean, string, number, Flag} from './flags'
```

Comment:
I'd rather see this exporting all of the flags in one object. Though I'm not sure how to write it with import/export syntax. It would be this with common.js:

```js
exports.flags = require('./flags')
```